contracts/src/access/control.rs

```diff
@@ -412,26 +412,26 @@ mod tests {
     }
 
     #[motsu::test]
-    fn default_role_is_default_admin(contract: AccessControl) {
+    fn default_admin_role_success(contract: AccessControl) {
```

Comment:
```suggestion
    fn get_role_admin_returns_default_admin_for_regular_role(
        contract: AccessControl,
    ) {
```
Don't forget the requirements:

> `function_name_description` for successful tests.

`function_name` is the name of the function that is being tested, and this should apply to getters as well. 

**Double check that all the other test names follow this convention.**